src/runtime.jl

```diff
@@ -92,3 +92,24 @@ function get_julia_libpaths()
     end
     return JULIA_LIBDIRS
 end
+
+@static if VERSION >= v"1.6.0" && libc(HostPlatform()) == "musl"
```

Comment:
At this point we can do #31 as well? :upside_down_face:

Reply:
Heh, sure.  We can do both for now.